huxley/www/js/components/DelegationAttendanceRow.js

```diff
@@ -11,8 +11,17 @@ var DelegationAttendanceRow = React.createClass({
   propTypes: {
     onChange: React.PropTypes.func,
     countryName: React.PropTypes.string,
-    countryID: React.PropTypes.string,
-    delegates: React.PropTypes.array,
+    assignmentID: React.PropTypes.number,
+    attendance: React.PropTypes.object,
+  },
+
+  shouldComponentUpdate(nextProps, nextState) {
```

Comment:
This is a dangerous `shouldComponentUpdate`, because it won't re-render the component if the `assignmentID` changes. In theory, it never should, but this component cannot make any guarantees as to whether it'll be used correctly.